app/models/solidus_avatax_certified/line.rb

```diff
@@ -122,24 +122,14 @@ def return_item_line(line_item, quantity, amount)
         quantity: quantity,
         amount: -amount.to_f,
         addresses: {
-          shipFrom: get_stock_location(line_item),
+          shipFrom: ship_from_address('return_line', line_item),
           shipTo: ship_to
         }
       }.merge(base_line_hash)
 
       return_item_line.merge(tax_override_hash(amount))
     end
 
-    def get_stock_location(li)
-      inventory_units = li.inventory_units
-
-      return default_ship_from if inventory_units.blank?
```

Comment:
We need to get rid of `default_ship_from` method. And use company address in `refund_line` instead.
Let's confirm that from Ben and Don but that is how it should be I think.